code/modules/client/preference_setup/loadout/loadout_head.dm

```diff
@@ -235,6 +235,11 @@
 	display_name = "kitty ears"
 	path = /obj/item/clothing/head/kitty
 
+/datum/gear/head/halo
```

Comment:
Don't forget // chompedits for files that aren't ch-specific.

Reply:
OKAY, MOM